docs/pages/enterprise/workflow/ssh-approval-pagerduty.mdx

```diff
@@ -117,10 +76,18 @@ $ sudo mv teleport-pagerduty.toml /etc
 
 After generating the config, edit it as follows:
 
-```toml
-# Example PagerDuty config file
-(!examples/resources/plugins/teleport-pagerduty.toml!)
+<Tabs>
+<TabItem scopes={["oss", "enterprise"]} label="Self Hosted">
```

Comment:
Should we standardize on "self-hosted" with a dash? It's written this way in other pages above.
```suggestion
<TabItem scopes={["oss", "enterprise"]} label="Self-Hosted">
```